psci/PSCi.hs

```diff
@@ -487,7 +487,11 @@ handleBrowse moduleName = do
     Left errs -> printErrors errs
     Right env' ->
       if moduleName `notElem` (nub . map ((\ (P.Module _ _ modName _ _ ) -> modName) . snd)) (psciLoadedModules st)
-        then PSCI $ outputStrLn $ "Module '" ++ N.runModuleName moduleName ++ "' is not valid."
+        then case Just moduleName `lookup` (nub . map ( \(modName, _, mayQuaName) -> (mayQuaName, modName))) (psciImportedModules st) of
+          Nothing ->
+            PSCI $ outputStrLn $ "Module '" ++ N.runModuleName moduleName ++ "' is not valid."
+          Just qualifiedName ->
+            handleBrowse qualifiedName
```

Comment:
We can just copy and run the `if` on line 489 again instead, would that be better?